src/media/mic.rs

```diff
@@ -142,6 +144,15 @@ impl Iterator for Microphone {
     }
 }
 
+/// Specify the configuration for the [`Microphone`] constructor.
+///
+/// Field is optional and will be set to hardcoded value.
+#[derive(Clone, Debug, Default)]
+pub struct MicrophoneInitOptions {
```

Comment:
Maybe the name `MicrophoneOptions` would be more consistent with `AudioContextOptions`